SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/config.h.in

```diff
@@ -19,16 +19,16 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#ifndef SOFABASECOLLISION_CONFIG_H
-#define SOFABASECOLLISION_CONFIG_H
+#pragma once
 
-#include <SofaBase/config.h>
+#include <sofa/config.h>
+#include <sofa/config/sharedlibrary_defines.h>
```

Comment:
what is this header?

Reply:
This one ?
`
#include <sofa/config/sharedlibrary_defines.h>
`
It defines the macros SOFA_EXPORT_DYNAMIC_LIBRARY and SOFA_IMPORT_DYNAMIC_LIBRARY